libafl/Cargo.toml

```diff
@@ -30,6 +30,8 @@ rustc-args = ["--cfg", "docsrs"]
 default = [
   "std",
   "derive",
+  "tcp_manager",
+  "multi_machine",
```

Comment:
Just a reminder to remove this again when the PR is done

Reply:
ok